CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ include(${_project_options_SOURCE_DIR}/Index.cmake)
 # Define the features of the project
 include("./Features.cmake")
 
+# enable cross-compiling: - should be called before run_vcpkg()
+if(ENABLE_CROSS_COMPILING)
+  enable_cross_compiler()
```

Comment:
Does this need any parameters?

Reply:
> Does this need any parameters?
No, the fallback is Parameters (or env. variables). 

https://github.com/aminya/cpp_vcpkg_project/pull/15/files#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425e
```yml
build_cross_mingw:
    - task: build_template
      vars:
        FEATURE_TESTS: OFF
        CMAKE_BUILD_TYPE: Release
        CONFIGURE_FLAGS: -DENABLE_CROSS_COMPILING:BOOL=ON -DDEFAULT_TRIPLET=x64-mingw-dynamic
```

Reply:
Ok, thanks! I think we can add this to the documentation:
https://github.com/aminya/project_options/blob/main/docs/src/enable_cross_compiler.md